test/lms.inc

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "lms_common.inc"
+
+typedef struct LMS_ACVP_TEST_DATA_st {
+    const unsigned char *pub;
+    size_t publen;
+    const unsigned char *priv;
+    size_t privlen;
+    const unsigned char *msg;
+    size_t msglen;
+    const unsigned char *sig;
+    size_t siglen;
+} LMS_ACVP_TEST_DATA;
+
+/*
+ * The data for HSS with a single level is almost identical
```

Comment:
This comment doesn't belong in this PR.

Reply:
The data in the inc file is actually HSS data, so it makes sense. This will get resolved in the HSS part.